Northstar.Client/mod.json

```
@@ -117,11 +117,22 @@
 			"RunOn": "UI"
 		},
 		{
+			"Path": "ui/ms_slider.nut",
+			"RunOn": "UI"
+		},
+		{
+			"Path": "ui/menu_mod_settings.nut",
+			"RunOn": "UI",
+			"UICallback": {
+				"Before": "AddModSettingsMenu"
+      }
+    },
+    {
 			"Path": "ui/ui_mouse_capture.nut",
 			"RunOn": "UI"
 		}
 	],
 	"Localisation": [
 		"resource/northstar_client_localisation_%language%.txt"
 	]
-}
+}
```

```suggestion
}

```
end your files with a newline you heathens